test(jans-auth-server): fix TokenExchangeServiceTest (native sso)

**Describe the bug**

```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/server/src/test/java/io/jans/as/server/token/ws/rs/TokenExchangeServiceTest.java:[19,0] error: cannot find symbol
[ERROR]  class TestUtil
/home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/server/src/test/java/io/jans/as/server/token/ws/rs/TokenExchangeServiceTest.java:[20,0] error: cannot find symbol
[ERROR]  class TestUtil
/home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/server/src/test/java/io/jans/as/server/token/ws/rs/TokenExchangeServiceTest.java:[59,8] error: cannot find symbol
[ERROR]  class TokenExchangeServiceTest
/home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/server/src/test/java/io/jans/as/server/token/ws/rs/TokenExchangeServiceTest.java:[71,8] error: cannot find symbol
[ERROR]  class TokenExchangeServiceTest
/home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/server/src/test/java/io/jans/as/server/token/ws/rs/TokenExchangeServiceTest.java:[83,8] error: cannot find symbol
[ERROR]  class TokenExchangeServiceTest
```

**To Reproduce**
Run build.

**Expected behavior**
Test must pass.

--- a/jans-auth-server/server/src/test/java/io/jans/as/server/util/TestUtil.java
+++ b/jans-auth-server/server/src/test/java/io/jans/as/server/util/TestUtil.java
@@ -3,10 +3,10 @@ package io.jans.as.server.util;
 import io.jans.as.client.RegisterResponse;
 import io.jans.as.model.error.ErrorResponse;
 import jakarta.ws.rs.core.Response;
+import org.apache.commons.lang3.StringUtils;
 
 import static io.jans.as.model.uma.TestUtil.assertNotBlank;
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.*;
 
 /**
  * @author yuriyz
@@ -14,6 +14,14 @@ import static org.testng.Assert.assertNotNull;
 public class TestUtil {
 
     private TestUtil() {
+    }
+
+    public static void assertEmpty(String s) {
+        assertTrue(StringUtils.isBlank(s));
+    }
+
+    public static void assertNotEmpty(String s) {
+        assertTrue(StringUtils.isNotBlank(s));
     }
 
     public static void assert_(RegisterResponse response) {

